publish.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
+apply plugin: 'java-library'
```

Comment:
I believe that `java-library` is not compatible with the `java` plugin, so you have to pick one.

Reply:
removed!